src/main/java/org/broadinstitute/hellbender/engine/AssemblyRegion.java

```diff
@@ -438,7 +439,7 @@ public SimpleInterval getReadSpanLoc() {
      * @param genomeLoc a non-null genome loc indicating the base span of the bp we'd like to get the reference for
      * @return a non-null array of bytes holding the reference bases in referenceReader
      */
-    private static byte[] getReference(final IndexedFastaSequenceFile referenceReader, final int padding, final SimpleInterval genomeLoc) {
+    private static byte[] getReference(final ReferenceSequenceFile referenceReader, final int padding, final SimpleInterval genomeLoc) {
```

Comment:
name of the `referenceReader` parameter seems weird given that the type is called `ReferenceSequenceFile`